### DISCUSSION TITLE
Tabbing over a semi-colon

### DISCUSSION THREAD

**Discussion by jtyler76** [abecodes/tabout.nvim#52] [Q&A]
Is there a way to configure this plugin such that you can TAB over a semicolon like so?

`printf("Hello World!\n")|; => TAB => printf("Hello World!\n");|`

**Comment by abecodes** [comment]
Hi @jtyler76 😊 

Since the plugin is heavily relaying on treesitter: no. Thats not a valid node combination (opening with `(` and closing with `;`).
You could try the `tabout-md` branch and add the combo to the config (removing the `()` one), that may work if you are lucky 🤞
